modules/op-about-workspace.adoc

```diff
@@ -0,0 +1,96 @@
+// This module is included in the following assembly:
+//
+// *openshift_pipelines/creating-applications-with-cicd-pipelines.adoc
+
+[id="about-workspaces_{context}"]
+= About Workspaces
+
+Workspaces declare shared storage volumes that a Task in a Pipeline needs at runtime. Instead of specifying the actual location of the volumes, Workspaces enable the Task authors to declare the filesystem or parts of the filesystem that would be required at runtime. The onus is on the Task or Pipeline user to provide the specific details of the volume that is mounted into that Workspace in a TaskRun or a PipelineRun. This separation of volume declaration from runtime storage volumes makes the Tasks reusable, flexible and independent of the user environment.
```

Comment:
The onus -> The bones :stuck_out_tongue_closed_eyes:

Reply:
replaced and reworded :)